pages/k8s/charm-etcd.md

```diff
@@ -174,27 +174,13 @@ juju add-storage etcd/0 data=cinder,10G
 ### Restore
 
 Allows the operator to restore the data from a cluster-data snapshot. This
-comes with caveats and a very specific path to restore a cluster:
+action is supported on both single member and cluster deployments. However
+it must be always executed on the leader unit.
```

Comment:
Thanks - which version of the etcd charm is this good from?

Reply:
Hmm, good question. I'm not exactly sure what's the process there, but this change is tied to this PR https://github.com/charmed-kubernetes/layer-etcd/pull/183 , So my guess is that this change is good from the moment the code PR gets merged to master?

Reply:
I think there is also a disconnect between that being merged and the new rev of the charm making it to the store. 
It doesn't normally matter too much, but I don't want this to get pulled in to the 1.20ck1 bugfix release update for docs if it isn't actually there. I'll watch that other PR and see what actually happens before I merge this (thanks for the contribution though!)
We are working on a way of keeping the docs in the repo with the charm so this doesn't have to get so messy in future!

Reply:
This should go out with 1.20+ck2

Reply:
1.20+ck2 never happened, and the fixes were rolled into 1.21. So, this should now go out with 1.21.